projects/angular/src/forms/datepicker/datepicker-view-manager.ts

```diff
@@ -17,13 +18,20 @@ import { ViewManagerService } from './providers/view-manager.service';
   providers: [DatepickerFocusService],
   host: {
     '[class.datepicker]': 'true',
+    '[class.has-action-buttons]': 'hasActionButtons',
     '[attr.aria-modal]': 'true',
     '[attr.aria-label]': 'commonStrings.keys.datepickerDialogLabel',
     role: 'dialog',
   },
 })
 export class ClrDatepickerViewManager {
-  constructor(public commonStrings: ClrCommonStringsService, private viewManagerService: ViewManagerService) {}
+  hasActionButtons = this.dateNavigationService.hasActionButtons;
```

Comment:
We don't need that as variable
```suggestion
  protected get hasActionButtons() {
     return this.dateNavigationService.hasActionButtons;
  }
```